package.json

```diff
@@ -2897,4 +2909,4 @@
 		"jsonc-parser": "^2.3.0",
 		"lodash": "^4.17.19"
 	}
-}
+}
```

Comment:
Undo the removal of the new line character

Reply:
Ah, this was VSCode! Will revert